src/view/QuestionsList.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+//nu-mi trebe user aicea, ci in presenter si il pot trimite din smart component
+//da cand creez un nou question imi trebuie si useru, da pot sa-l adaug dupa
+//da-l in ma0sa, il harcodez
+
+import 'bulma/css/bulma.css';
+
+
+
+
+const QuestionsList = ({ questions, onCreateQuestion, searchTitle, searchTag, onChange }) => (
+
+            <div className={"column"}>
+                    <section className="hero is-fullheight is-dark is-bold">
```

Comment:
This file has very strange indenting (8 spaces). I guess this is because of copy-pasting from the bulma docs?